src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -222,6 +272,30 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
         ***********************************************************************/
 
         public RequestId request_id;
+
+        /***********************************************************************
+
+            Helper function to properly serialize user-specified params for the
+            request into the `user_params` field. An intermediary serialzation
+            buffer is used to safely convert from the const params provided by
+            the user to the mutable params required internally (in order to be
+            able to deserialize into a Context instance).
+
+            Params:
+                params = user-specified request parameters to store
+                serialize_buffer = intermediary serialization buffer
+
+        ***********************************************************************/
+
+        public void setUserSpecifiedParams ( Const!(UserSpecifiedParams) params,
+            ref ubyte[] serialize_buffer )
+        {
+            auto serialized = Serializer.serialize(params, serialize_buffer);
+            enableStomping(serialized);
```

Comment:
It is a bit weird that `enableStomping` is required here, `serialize_buffer` is not resized in any other place, right?